typedb-src/modules/ROOT/pages/introduction.adoc

```diff
@@ -2,21 +2,22 @@
 :keywords: typedb, database, documentation, introduction, overview
 :longTailKeywords: typedb introduction, typedb overview, learn typedb, learn typeql, typedb schema, typedb data model
 :pageTitle: TypeDB introduction
-:summary: A birds-eye view of TypeDB.
+:summary: A bird's-eye view of TypeDB.
 
 == Summary
 
-TypeDB is a strongly typed database that is unique in many ways. TypeDB:
+TypeDB is a transactional (https://en.wikipedia.org/wiki/Online_transaction_processing[OLTP,window=_blank])
+strongly typed DBMS with xref:development/connect.adoc#_acid_guarantees[ACID] guarantees that is unique in many ways:
 
-* has its own special xref:fundamentals/types.adoc[static type system]
-* uses an xref:fundamentals/types.adoc#_root[Entity-Relationship model], extended with
-xref:fundamentals/types.adoc#_attribute_types[Attributes] and xref:fundamentals/inference.adoc#_rules[Rules]
-// - #todo add link to the types article
-* supports subtyping and inheritance like OOP languages
-* uses its own declarative query language, xref:typeql:ROOT:overview.adoc[TypeQL,window=_blank]
-* is a transactional (https://en.wikipedia.org/wiki/Online_transaction_processing[OLTP,window=_blank]) DBMS with
-xref:development/connect.adoc#_acid_guarantees[ACID] guarantees
-* separates data into xref:development/connect.adoc#_databases[databases]
+* uses its own declarative query language: xref:typeql:ROOT:overview.adoc[TypeQL,window=_blank]
```

Comment:
Each of these needs a subject, either "it" or "TypeDB;" or restructure the predicates.